config/rbac/role.yaml

```diff
@@ -6,6 +6,18 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - '*'
+- apiGroups:
+  - cloudcredential.openshift.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
```

Comment:
Same here, I think that we really should only ask for GET and Update/Patch/Delete for our specific one? Can we have a standard name and we can reduce the scope?

Reply:
Done, reduced the scope and updated the perms.